### ISSUE TITLE
Does Zuul2.x support tcp connections?

### ISSUE THREAD

**Issue by xcbeyond** [Netflix/zuul#493]
Hi！
    Does Zuul2.x support tcp and https connections? 
    proxy protocol? give me an example? thanks you very much.

**Comment by artgon** [comment]
Yes, please look at the [wiki](https://github.com/Netflix/zuul/wiki).

**Comment by xcbeyond** [comment]
@artgon thank you for your reply.
I've seen [Proxy Protocol](https://github.com/Netflix/zuul/wiki/Core-Features#proxy-protocol) at the wiki.But, I don't know how to use it. Can you give me a detailed examples?

**Comment by xcbeyond** [comment]
@artgon Can you give me a detailed tcp connections examples? thank you very much!

**Comment by artgon** [comment]
@xcbeyond what's missing from the wiki? What do you want more explanation about?

**Comment by xcbeyond** [comment]
@artgon yes,gave me a detailed tcp connections examples in the wiki.

**Comment by artgon** [comment]
I thought you were asking about proxy protocol? Zuul doesn't support raw TCP request proxying. It requires the HTTP request/response paradigm.
